scripts/server

```diff
@@ -12,17 +12,26 @@ Starts servers using docker-compose
 "
 }
 
-
 if [ "${BASH_SOURCE[0]}" = "${0}" ]
 then
     if [ "${1:-}" = "--help" ]
     then
         usage
+    elif [ "$1" = "--no-airflow" ]
+    then
+        docker-compose -f "${DIR}/../docker-compose.yml" up \
```

Comment:
Instead of enumerating each service here, I think it would make sense to split Redis and the Airflow services out into a `docker-compose.airflow.yml`. Then, with `--with-airflow` condition can just `docker-compose up` with both YAML files supplied by `-f`.

Downside is that changes will have to be made to various `scripts` so that commands requiring Airflow and Redis will need the additional `-f docker-compose.airflow.yml`.